rest/data/task.go

```diff
@@ -37,6 +37,20 @@ func (tc *DBTaskConnector) FindTaskById(taskId string) (*task.Task, error) {
 	return t, nil
 }
 
+func (tc *DBTaskConnector) FindTaskByIdAndExecution(taskId string, execution int) (*task.Task, error) {
+	t, err := task.FindByIdExecution(taskId, &execution)
+	if err != nil {
+		return nil, err
+	}
+	if t == nil {
+		return nil, gimlet.ErrorResponse{
+			StatusCode: http.StatusNotFound,
+			Message:    fmt.Sprintf("task with id %s not found", taskId),
```

Comment:
nit: wrap formatting with `` for readability (i.e. `%s`)

Reply:
Some of these comments say done but I'm thinking you forgot to push them; I resolved the ones that actually look done.

Reply:
I think it's because i used single quotes instead of a backtick 
fixed